src/libraries/icubmod/embObjLib/ethBoards.cpp

```diff
@@ -25,6 +25,8 @@
 // --------------------------------------------------------------------------------------------------------------------
 
 #include "ethBoards.h"
+#include <yarp/os/LogStream.h>
+#include <yarp/os/Log.h>
```

Comment:
Why was this necessary?

Reply:
Yeh, right, I've added those 2 lines in ethBoards.cpp for debug reason. I'll remove them from this PR.